### PR TITLE
Added healthcheck route to app.js. Changed spelling on readme.

### PR DESCRIPTION
**Pull request by jwpf100**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,9 @@
 This project is an open source Job Board. It's primary purpose is to serve as a
 training exercise.
 
-
 ## Getting Started
 
-1. Clone the repo `git vlone git@github.com:MattGibney/JobBoard.git`
+1. Clone the repo `git clone https://github.com/MattGibney/JobBoard.git`
```
Comment (MattGibney) [inline README.md:8]: Good spot on the typo.

I'd rather not change the repo URL though, It's intentionally like this. Using SSH is much better than HTTPS (In my opinion).

File: src/app.js [hunk 2]
```diff
@@ -6,6 +6,10 @@ app.get('/', (req, res) => {
   res.send({ version: config.version });
 });
 
+app.get('/healthcheck', (req, res) => {
+  res.send({ version: config.version });
+});
```
Comment (MattGibney) [inline src/app.js:11]: 👍🏻

**Review by MattGibney — CHANGES_REQUESTED** [review]
- [ ] Change the URL back to SSH variant

File: README.md (hunk 1 above)
Comment (jwpf100) [reply README.md:8]: No problem, I'll take your lead on that.  I didn't realise that was what the difference was so good to know.

**Review by jwpf100 — COMMENTED** [review]
Ok, I think that should be complete.

**Pull request merged**